package.json

```diff
@@ -39,27 +43,63 @@
 		"write",
 		"cache"
 	],
+	"ava": {
+		"files": [
+			"test/*"
+		],
+		"timeout": "1m",
+		"typescript": {
+			"rewritePaths": {
+				"test/": "dist/test/"
+			}
+		}
+	},
+	"xo": {
+		"extends": "xo-typescript",
+		"extensions": [
+			"ts"
+		]
+	},
```

Comment:
XO now supports TS built-in, so you can upgrade and drop this. See: https://github.com/xojs/xo/releases/tag/v0.27.0